### PR TITLE
Switch infrastructure to Arcade

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "sdk": {
-    "version": "2.1.300"
+  "tools": {
+    "dotnet": "2.1.401"
   },
   "msbuild-sdks": {
-    "RoslynTools.RepoToolset": "1.0.0-beta2-63103-01"
+    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.18515.2"
```
Comment (ctaggart) [inline global.json]: What is Arcade?

File: eng/Versions.props [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     <!-- This repo version -->
     <VersionPrefix>1.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
```
Comment (ctaggart) [inline eng/Versions.props]: Is `PreReleaseVersionLabel` a new property?

File: global.json (hunk 1 above)
Comment (tmat) [reply global.json]: https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md

Comment (tmat) [reply global.json]: Common infrastructure for .NET Core repos.

File: eng/Versions.props (hunk 2 above)
Comment (tmat) [reply eng/Versions.props]: It's property defined by Arcade.

**Pull request merged**